src/pattern_count.rs

```diff
@@ -31,4 +31,31 @@ pub fn frequent_words(text: &str, k: usize) -> Vec<&str> {
     }
 
     return frequent;
-}
\ No newline at end of file
+}
+
+
+//pub fn reverse_string(text: &str) -> &str {
+//    let mut reverse = &text.chars().rev();
+//    return reverse;
+//}
+
+
+//week2
+
+
+
+pub fn skew(text: &str) -> Vec<&str> {
+    let mut text = "CATGGGCATCGGCCATACGCC";
+   // let mut skew_gc = vec![ ];
+    let mut count_gc = vec![0 ; text.len()];
+    for i in 0..text.len() {
+        if &text[i] == "G" {
+            count_gc == count_gc += 1;
```

Comment:
This is syntactically incorrect `==` is a comparison operator, not assignation.
maybe you should do something like `count_gc[i + 1] = count_gc[i] + 1`?